core/src/main/java/org/bitcoinj/wallet/KeyChainGroupStructure.java

```diff
@@ -16,14 +16,54 @@
 
 package org.bitcoinj.wallet;
 
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.crypto.ChildNumber;
 import org.bitcoinj.crypto.HDPath;
 import org.bitcoinj.script.Script;
 
 /** Defines a structure for hierarchical deterministic wallets. */
 public interface KeyChainGroupStructure {
+    ChildNumber PURPOSE_BIP44 = new ChildNumber(44, true);  // P2PKH
+    ChildNumber PURPOSE_BIP49 = new ChildNumber(49, true);  // P2WPKH-nested-in-P2SH
+    ChildNumber PURPOSE_BIP84 = new ChildNumber(84, true);  // P2WPKH
+
+    ChildNumber COINTYPE_BTC = new ChildNumber(0, true);
+    ChildNumber COINTYPE_TBTC = new ChildNumber(1, true);
+    ChildNumber COINTYPE_LTC = new ChildNumber(2, true);
```

Comment:
Support for litecoin in bitcoinj? Does this make sense?

Reply:
I recall the Litecoin value being in the BIP, but it's not there currently. SLIP-44 defines all the other coin types.

I put it there just to remind me that other coin types can go there. 

We can remove it if you want.

Reply:
Shall I leave it or remove it?

Reply:
I reluctantly agree with that strategy, but I sometimes think might get more upstream fixes if they could use our JAR directly.

Reply:
`COINTYPE_LTC` removed in new force-push.